fre/make/gfdlfremake/yamlfre.py

```diff
@@ -176,8 +177,8 @@ def __init__(self,combinedyaml,v):
         #self.freyaml.update(self.platformsyaml)
 
         ## VALIDATION OF COMBINED YAML FOR COMPILATION
-        fremake_package_dir = os.path.dirname(os.path.abspath(__file__))
-        schema_path = os.path.join(fremake_package_dir, 'schema.json')
+        fremake_package_dir = Path(__file__).resolve().parents[2]
+        schema_path = os.path.join(fremake_package_dir, 'gfdl_msd_schemas', 'FRE', 'fre_make.json')
```

Comment:
not sure the `parents` thing is worth it here, `fre/make` is pretty unlikely to ever change in location.

Reply:
I just saw it as another easy way to define absolute paths / find the directory names ... I'll look into if it can introduce issues

Reply:
I think I'll keep this one here. It finds the right schema location in reference to this file itself, imo, in a simple way